erpnext/www/lms/course.py

```diff
@@ -20,7 +20,19 @@ def get_context(context):
 	context.topics = course.get_topics()
 	context.has_access =  utils.allowed_program_access(context.program)
 	context.progress = get_topic_progress(context.topics, course, context.program)
+	context.total_progress = calculate_progress(context.topics,context.progress)
+
 
 def get_topic_progress(topics, course, program):
 	progress = {topic.name: utils.get_topic_progress(topic, course.name, program) for topic in topics}
 	return progress
+
+def calculate_progress(topics,progress):
```

Comment:
please put down comments on what this code block does - it will improve the quality of our codebase and help people understand what they are reading.

Reply:
@rvpasari 
Basically, the calculate_progress() function calculates progress for courses, topics, and topic contents. 
This function simply calculates progress based upon how many total_courses/total_topics/total_topics_content are available in a particular program and how many courses/topics/topics_content are completed.
In my latest commit, you can see a proper name for each progress.